module/documents/macro.mjs

```diff
@@ -80,12 +80,15 @@ function getMacroTarget(name, documentType) {
 /* -------------------------------------------- */
 
 /**
- * Trigger an item to roll when a macro is clicked.
+ * Trigger an item to be used when a macro is clicked.
  * @param {string} itemName                Name of the item on the selected actor to trigger.
- * @returns {Promise<ChatMessage|object>}  Roll result.
+ * @param {string} [activityName]          Name of a specific activity on the item to trigger.
+ * @returns {Promise<ChatMessage|object>}  Usage result.
  */
-export function rollItem(itemName) {
-  return getMacroTarget(itemName, "Item")?.use();
+export function rollItem(itemName, activityName) {
```

Comment:
It's conceivable that we'll add more options to the item enricher in the future so maybe we should move this parameter into an `options` object pre-emptively, since it is public API.